README.md

```diff
@@ -206,3 +206,8 @@ Some information of the basic settings will be written to the Clusterfile and st
 ```shell script
 sealer delete -f /root/.sealer/my-cluster/Clusterfile
 ```
+
+# Developing Sealer
+
+* [contributing guide](./CONTRIBUTIONG.md)
+* [贡献文档](./docs/contributing_zh.md)
```

Comment:
How about also put the CONTRIBUTIONG_ZH.md at the root dir of the project?

Reply:
I'am afraid there will be too many file in root dir.